pkg/alertmanager/alertmanager.go

```diff
@@ -97,9 +104,16 @@ func New(cfg *Config, reg *prometheus.Registry) (*Alertmanager, error) {
 		stop:      make(chan struct{}),
 		active:    false,
 		activeMtx: sync.Mutex{},
+		configHashMetric: prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Who adds the `user` label to this metric, when it gets exported?

Reply:
See: https://github.com/cortexproject/cortex/blob/master/pkg/alertmanager/multitenant.go#L442-L460